.drone.star

```diff
@@ -552,29 +495,30 @@ def uiTests(ctx):
     for item in default:
         params[item] = config["uiTests"][item] if item in config["uiTests"] else default[item]
 
-    suiteNames = config["uiTests"]["suites"].keys()
-
     if len(params["debugSuites"]) != 0:
         suiteNames = params["debugSuites"]
-
-    return [uiTestPipeline(ctx, suiteName, params["filterTags"]) for suiteName in suiteNames]
+        return [uiTestPipeline(ctx, suiteName, params["filterTags"]) for suiteName in suiteNames]
+    return [uiTestPipeline(ctx, None, params["filterTags"])]
```

Comment:
IMO https://drone.owncloud.com/owncloud/ocis/5182/56/2 does not like the duplicated `return`

Reply:
hmm ci lint reports single-quoted strings